README.md

```diff
@@ -10,44 +10,71 @@
 [![Docs](https://img.shields.io/badge/Docs-Booster-blue)](https://docs.booster.cloud)
 ---
 
-## Disclaimer
+# What is Booster Framework?
 
-Booster is still under heavy development, so non-backward-compatible changes might be introduced until we reach v1.0.0.
+The Booster Framework is a true serverless framework that provides a highly opinionated implementation of the CQRS and Event Sourcing patterns in Typescript. Using these patterns, you naturally write highly semantic, simple, and strongly typed code that puts into practice the ideas of Domain-Driven Design.
 
-Refer to [Release Notes](https://github.com/boostercloud/booster/releases) for more specific information about changes of each iteration.
+Booster extracts metadata from your code structure at deployment time, combining it with years worth of knowledge and experience from our team to build an optimal cloud environment to run your application at scale.
+‍
+Combining these features, Booster provides an unprecedented developer experience. On the one hand, it helps you write simpler code, defining your application in terms of commands, events, entities, and read models. On the other hand, you don't have to worry about the tremendous amount of low-level configuration details of conventional tools. You write highly semantic code, and if it compiles, it will run on the cloud.
 
-## The "Booster Way"
+Booster is designed with extensibility in mind. If there's still something you can't do with the default functionality, there are lower-level APIs that you can use to extend any layer of the framework with Rockets (The way we call Booster's plugins). With rockets, it's easy to add support for new cloud building blocks or pack functionality in highly reusable npm packages.
```

Comment:
We should figure out if we write `Rockets` or `rockets`

Reply:
I'd go for Rockets, as they're kind of a "brand"